src/operator/quantization/quantized_conv.cu

```diff
@@ -270,7 +270,7 @@ void QuantizedConvForwardGPU(const nnvm::NodeAttrs& attrs,
   const ConvolutionParam& param = nnvm::get<ConvolutionParam>(attrs.parsed);
   CHECK_EQ(param.kernel.ndim(), 2U)
     << "QuantizedConvForward<gpu> only supports 2D convolution for now";
-#if MXNET_USE_CUDNN == 1 && CUDNN_MAJOR >= 6
+#if MXNET_USE_CUDNN == 1 && CUDNN_MAJOR >= 6 && CUDA_VERSION >= 8000
```

Comment:
Does this need to have an else clause? What is the expected behavior if quantizedpoolforwadgpu is called with cuda version < 8 ?

Reply:
There will be an error message printed saying that otherwise is not supported. I have made it clearer on specifying cudnn and cuda versions.